package.json

```diff
@@ -12,6 +12,10 @@
     }
   ],
   "main": "index.js",
+  "files": [
```

Comment:
As we have constrict the `files` here, only distributed libraries and LICENSE/README will be distributed. The `src`, `scripts`, dev infrastructure `babel.config.js` will not be included in our npm package. Thus we obtain an optimal node_modules footprint.

Here is the result of `npm pack`:
```
npm notice
npm notice 📦  grapheme-splitter@1.0.4
npm notice === Tarball Contents ===
npm notice 1.1kB   package.json
npm notice 839B    index.d.ts
npm notice 145.1kB index.js
npm notice 1.1kB   LICENSE
npm notice 5.2kB   README.md
npm notice === Tarball Details ===
npm notice name:          grapheme-splitter
npm notice version:       1.0.4
npm notice filename:      grapheme-splitter-1.0.4.tgz
npm notice package size:  32.7 kB
npm notice unpacked size: 153.3 kB
npm notice shasum:        c3455c5317b8c40b340d7c7035b78edf11c561e7
npm notice integrity:     sha512-p+i2AbQ0PNq/T[...]3C7vTVdWa8gLQ==
npm notice total files:   5
```